CONTRIBUTING.rst

```diff
@@ -68,8 +68,18 @@ We recommend the following workflow:
    c. Document any user-facing changes in one of the ``/docs/`` files.
       Please use `one sentence per line`_.
 
-   d. Create a newsfragment in ``src/towncrier/newsfragments/`` describing the changes and containing information that is of interest to end-users.
+   d. Create a news fragment in ``src/towncrier/newsfragments/`` describing the changes and containing information that is of interest to end-users.
       Please use `one sentence per line`_ here, too.
```

Comment:
```suggestion
   d. Create a news fragment in ``src/towncrier/newsfragments/`` describing the changes and containing information that is of interest to end-users. That is a file which name consists out of the number of your PR, and the type of change that is introduced.
      Please use `one sentence per line`_ here, too.
```
I'd describe this even a little more.

Reply:
You do realize that this _is_ the Towncrier project, right? 😇 I think this is one of the few places where we don't have to explain what a news fragment is. More text is not necessarily better.